### PR TITLE
Adding test for "Accept-Ranges: none" scenario with partial downloads

### PR DESCRIPTION
**Pull request by travishathaway**
### Description

Fixes: #13671 

### Checklist - did you ...

- [ ] ~Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?~
     
- [x] Add / update necessary tests?
- [ ] ~Add / update outdated documentation?~

### CODE REVIEW

File: tests/fixtures_jlap.py [hunk 1]
```diff
@@ -53,6 +53,17 @@ def download_file(subdir, name):
...
 
+@app.route("/none-accept-ranges/")
+def none_accept_ranges():
+    """
+    Returns an empty request with "Accept-Ranges" set to "none"
+    """
+    response = flask.Response("test content test content test content")
+    response.headers["Accept-Ranges"] = "none"
+
+    return response
```
Comment (dholth) [inline tests/fixtures_jlap.py:64]: This may work. We will have to check that flask ignores the range request headers, and returns all the bytes even if we ask for only some of the bytes.
The test suite also contains a `http_test_server` fixture based on Python's `simplehttpserver`. It wouldn't return `accept-ranges: none`, which would be OK since we ignore that response header, but it may ignore the range request headers.

**Review by dholth — APPROVED** [review]
Thanks!
